src/omnisharp/options.ts

```diff
@@ -35,6 +35,18 @@ export class Options {
         public enableImportCompletion: boolean,
         public enableAsyncCompletion: boolean,
         public useSemanticHighlighting: boolean,
+        public enableForParameters: boolean,
```

Comment:
Would you be open to renaming these all with an `inlayHints` prefix? ala `inlayHintsEnabledForParameters`